src/components/Login/index.js

```diff
@@ -0,0 +1,67 @@
+import styled from "styled-components";
+import PropTypes from "prop-types";
+import isDefined from "@codewell/is-defined";
+import { useEffect, useState } from "react";
+import axios from "axios";
+
+const Wrapper = styled.div`
+  display: flex;
+  flex-direction: column;
+`;
+
+export const Login = ({ onChange, endpoint, ...props }) => {
+  const [value, setValue] = useState({});
```

Comment:
`state` and `setState`? This represents the component state with different state values

Reply:
Signal the intended use of the state by setting defaults
```
const [value, setValue] = useState({
  user: null,
  loggedIn: false,
  error: null,
});
```